ImmuDB4Net/ImmuStateSerde.cs

```diff
@@ -35,26 +34,13 @@ public class ImmuStateSerde
         if (session == null)
             return null;
         string contents = File.ReadAllText(fileName);
-        var deserialized = JsonSerializer.Deserialize<Dictionary<string, ImmuState>>(contents);
```

Comment:
Wouldn't it be easier to move the json serialization / deserialization code directly to the fille state handler?

If you think serde is still a valid addition, then the `Read` method should not get the `database` nor `session` parameters.

Reply:
it's easier indeed just to get rid of immustateserde. I have just removed it.